src/nvim/eval.lua

```diff
@@ -100,19 +100,19 @@ return {
     deletebufline={args={2,3}, base=1},
     dictwatcheradd={args=3},
     dictwatcherdel={args=3},
-    did_filetype={},
+    did_filetype={fast=true},
     diff_filler={args=1, base=1},
     diff_hlID={args=2, base=1},
     digraph_get={args=1, base=1},
     digraph_getlist={args={0, 1}, base=1},
     digraph_set={args=2, base=1},
     digraph_setlist={args=1, base=1},
     empty={args=1, base=1},
-    environ={},
-    escape={args=2, base=1},
+    environ={fast=true},
+    escape={args=2, base=1, fast=true},
     eval={args=1, base=1},
     eventhandler={},
-    executable={args=1, base=1},
+    executable={args=1, base=1, fast=true},
     execute={args={1, 2}, base=1},
     exepath={args=1, base=1},
     exists={args=1, base=1},
```

Comment:
`exists` would also be useful, but I'm not sure what works during fast events. Exists checks for files, but also for functions, env vars, commands and autocmds.

Reply:
For now, I've only added functions I'm confident are ok which basically just includes stuff that doesn't depend on any nvim state.

I have no idea if `exists` is ok in fast.

Reply:
Yeah, I agree. Would be hesitant to add that one. Also no idea if it's safe tbh